core/src/main/java/org/geysermc/geyser/entity/EntityDefinitions.java

```diff
@@ -894,6 +895,13 @@ public final class EntityDefinitions {
                     .type(EntityType.TRADER_LLAMA)
                     .identifier("minecraft:llama")
                     .build();
+            CAMEL = EntityDefinition.inherited(CamelEntity::new, abstractHorseEntityBase)
```

Comment:
I add these in alphabetical order where possible. Put this above the DONKEY declaration.